cms/admin/placeholderadmin.py

```diff
@@ -579,24 +594,43 @@ def move_plugin(self, request):
                         return HttpResponseBadRequest(force_text(
                             _('parent must be in the same language as '
                               'plugin_language')))
-                    plugin.parent_id = parent.pk
-                    plugin.language = language
-                    plugin.save()
+                    plugin = plugin.update(refresh=True, parent=parent, **plugin_data)
```

Comment:
isn't ``parent_id=parent.pk`` quicker? I haven't checked how the ORM translates this in a while, though, I might be wrong